app/views/layouts/application.html.slim

```diff
@@ -2,15 +2,11 @@ doctype html
 html
   head
     title
-      | BudgetPlanner
+      | Budget Planner
+    = stylesheet_link_tag 'creative/manifest.css', media: 'all'
+    link[href="http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800" rel="stylesheet" type="text/css"]
+    link[href="http://fonts.googleapis.com/css?family=Merriweather:400,300,300italic,400italic,700,700italic,900,900italic" rel="stylesheet" type="text/css"]
```

Comment:
There is a helper in rails which helps to build such URLs – `stylesheet_link_tag`

Reply:
Also, I think there is a gem which makes font fetching easier. [This one](https://github.com/mmontossi/fonts-rails) for example

Reply:
The gem throws uninitialized constant MIME::CSS error, so I left just URL